src/tiler/styles.mss

```diff
@@ -65,14 +65,20 @@
   [stream_order=10] {
     line-color: @streamColor;
     line-width: 7.0 * @zoomBase;
+    line-join: round;
```

Comment:
Good idea. Consider moving these into `#drb_streams_50, #nhdflowline`, so that they don't have to be repeated for each zoom level.